index.html

```diff
@@ -173,126 +173,364 @@ <h2>
+              <a>unadjustedMovement</a> member is <code>true</code> the event 
+              coordinates must not be affected by the
+              underlying platform behaviors such as mouse acceleration. In other
+              words, UA must use the APIs provided by the underlying platform to
+              guarantee getting the raw events. If the <a>PointerLockOptions</a>' <a>unadjustedMovement</a> member is
+              <code>false</code> UA must rely on the default behavior of the
+              underlying platform regarding the mouse acceleration.
+            </dl>
+          </p>
+          <p>
+            In the <a>Pointer Lock State</a> the system mouse cursor must be hidden.
+            Movement and button presses of the mouse must not cause the window
+            to lose focus.
+          </p>
+          <p class="note">
+            Synthetic mouse events created by application script act the same
+            regardless of lock state.
+          </p>
+        </p>
+      </section>
+      <section>
+        <h3>
+          `PointerLockOptions` dictionary
+        </h3>
+        <pre class="idl">
+          dictionary PointerLockOptions {
+            boolean unadjustedMovement = false;
+          };
+        </pre>
+        <dl data-dfn-for="PointerLockOptions" data-link-for=
+        "PointerLockOptions">
+          <dt>
+            <dfn>PointerLockOptions</dfn> dictionary
+          </dt>
+          <dd>
+            <p>
+              The options dictionary to customize how the pointer behaves in
+              the locked mode.
+            </p>
+          </dd>
+          <dt>
+            <dfn>unadjustedMovement</dfn> member
+          </dt>
+          <dd>
+            <p>
+              If this value is set to `true`, then the pointer movements will
+              not be affected by the underlying platform modications such as
+              mouse accelaration.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3>
+          `"pointerlockchange"` and `"pointerlockerror"` Events
+        </h3>
+        <p class="note">
+          Two events are used to communicate pointer lock state change or an
+          error in changing state. They are named <dfn>pointerlockchange</dfn>
+          and <dfn>pointerlockerror</dfn>.
+        </p>
+        <p class="note">
+          Magnification software increases the size of content on the screen.
+          It uses the mouse to move the magnified point of focus around. When a
+          pointer lock is initiated, the magnification software needs to switch
+          to using the keyboard to move the magnified point of focus around
+          instead. When a pointerlockchange event is fired, web browsers
+          therefore need to make sure the event is communicated to assistive
+          technologies like screen magnifiers.
+        </p>
+      </section>
+      <section>
+        <h3><dfn>Exit Pointer Lock</dfn></h3>
```

Comment:
This algorithm is only referenced once in the document. Should it be called from more places than just exitPointerLock()? E.g., on document unloading?